.github/workflows/nodejs.yml

```
@@ -181,6 +181,10 @@ jobs:
       - name: Run typings tests
         run: npm run test:typescript
 
+  test-llhttp-integrity:
+    name: Ensure integrity of llhttp wasm files
+    uses: uzlopak/undici/.github/workflows/llhttp-wasm-integrity.yml@llhttp-integrity
```

this has to be modified to use undici main,

---

before merging, this has to be appied:

```suggestion
    uses: nodejs/undici/.github/workflows/llhttp-wasm-integrity.yml@main
```